package.json

```diff
@@ -47,8 +47,8 @@
     "compile:js": "npm run lint:js && mkdir -p public && browserify src/js/index.js > /tmp/bundle.js && mv /tmp/bundle.js public/bundle.js",
     "compile:css": "mkdir -p public && stylus src/css/index.styl -o public/index.css",
     "compile:html": "npm run compile:html:pages && npm run compile:html:templates",
-    "compile:html:pages": "mkdir -p public && pug src/html/pages/ --out public -O \"$(scripts/data_cache)\" --hierarchy",
-    "compile:html:templates": "src/templates \"$(scripts/data_cache)\"",
+    "compile:html:pages": "mkdir -p public && scripts/data_cache > /dev/null && pug src/html/pages/ --out public -O .data --hierarchy",
```

Comment:
@hhsnopek not sure if this feels like a hack to you, but basically I'm calling `data_cache` to make sure that the data is written to `.data` then I'm just passing in the file (which is an option to the pug compiler).

We can can do one of 3 things:
1. This
2. PR to pug to allow the -O flag to also be passed in as stdin
3. Create another file in scripts to compile pug, which we allow stdin to be passed and we use Pug in Javascript to compile everything out.

I think this strategy is more than sufficient, but I'd like to hear what you think.